lib/gitlib.sh

```diff
@@ -307,20 +307,65 @@ if ((FLAGS_gcb)); then
   gitlib::git_config_for_gcb || common::exit "Exiting..."
 fi
 
+##############################################################################
+# Publish an issue on github
+gitlib::create_issue() {
+  local repo="${1?expected repo for gitlib::create_issue}"
+  local title="${2?expected title for gitlib::create_issue}"
+  local body="${3?expected body for gitlib::create_issue}"
+  local milestone="${4:-null}"
+  local template data
+
+  # shellcheck disable=SC2016
+  template='{
+    "title": $title,
+    "body": $body,
+    "milestone": $milestone,
+  }'
+
+  data="$( jq \
+    --argjson milestone "$milestone" \
+    --arg     body      "$body" \
+    --arg     title     "$title" \
+    -c -n "$template"
+  )"
+
+  ${GHCURL} "${K8S_GITHUB_API_ROOT}/${repo}/issues" --data "$data"
+}
+
 ###############################################################################
-# Search for a matching release tracking issue
-# @param version - RELEASE_VERSION_PRIME
-# @param repo - org/repo
-# returns 1 if none found
-# prints most recent open issue matching
-gitlib::search_release_issue () {
-  local version=$1
-  local repo=$2
-  local issue
+# Post an issue for the publishing bot, to ask them to update their
+# configuration
+gitlib::create_publishing_bot_issue() {
+  local branch="$1"
+  local repo title body assign milestone
+
+  title="Update publishing-bot for ${branch}"
+  repo="k8s-release-robot/sig-release"
+  assign="<!-- no assignments -->"
+  milestone="v${branch#release-}"
+
+  if ((FLAGS_nomock)); then
+    repo="kubernetes/kubernetes"
+    assign="/assign @sttts @nikhita @dims"
```

Comment:
I am not sure if we'd want to assign specific people or if we have some GH team or such?

Reply:
I'd prefer a GH team over specific people.

Reply:
Created https://github.com/kubernetes/org/pull/558

Reply:
It seems `/assign`ing to an GH team does not work.
I tried to assign this very PR to `@kubernetes/bash-firefighters` but the @k8s-ci-robot didn't pick that up.

Reply:
Just tried using the UI, looks like GH doesn't support assigning teams. :-1: 

I'd be fine with cc'ing a team as well. But if others feel that assigning is more powerful, I'm ok with assigning specific people for now. Overall, just want to ensure that we don't create a SPOF-ish situation. :)

@sttts @dims thoughts?

Reply:
Just before we create the issue we could ask the GH API who is on that team and assign to them. I'll look into that.

Reply:
🐚 🔥 😼